addons/l10n_pl/i18n/l10n_pl.pot

```diff
@@ -195,13 +195,6 @@ msgstr ""
 msgid "Created on"
 msgstr ""
 
-#. module: l10n_pl
-#. odoo-python
-#: code:addons/l10n_pl/models/account_move.py:0
-#, python-format
-msgid "Credit notes can't have a total amount greater than the invoice's"
-msgstr ""
-
```

Comment:
You shouldn't remove translation terms in stable, even if they're not used anymore ;) 
(aslo in the *.po)

Reply:
Done, Ty